moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -53,12 +53,9 @@ bool pilz_industrial_motion_planner::computePoseIK(const planning_scene::Plannin
     ROS_ERROR_STREAM("Robot model has no planning group named as " << group_name);
     return false;
   }
-
-  if (!robot_model->getJointModelGroup(group_name)->canSetStateFromIK(link_name))
-  {
-    ROS_ERROR_STREAM("No valid IK solver exists for " << link_name << " in planning group " << group_name);
-    return false;
-  }
+  const moveit::core::JointModelGroup* jmg = robot_model->getJointModelGroup(group_name);
+  moveit::core::RobotState rstate = scene->getCurrentState();
+  rstate.setVariablePositions(seed);
```

Comment:
To minimize changes, I suggest to keep that block below the next if-block.

Reply:
Done :+1: